File: src/entities/aluno.ts
export class Aluno {
  public Id: string;
  public Nome: string;
  public selected: boolean;

  public static Create(id: string, nome: string) {
    var aluno = new Aluno();
    aluno.Id = id;
    aluno.Nome = nome;
    return aluno;
  }
}

File: backend/MatriculaServers.Core/Aluno.cs
namespace MatriculaServers.Core
{
    public class Aluno : Entity
    {
        public string Nome { get; set; }

        public Aluno()
        {
        }

        public Aluno(int id, string nome)
        {
            Id = id;
            Nome = nome;
        }

        public static Aluno Create(int id, string nome) 
            => new Aluno(id, nome);
    }
}
